public/index.html

```diff
@@ -98,6 +98,10 @@ <h1>Embeddable Widget Demo</h1>
       vulputate magna eros eu erat. Aliquam erat volutpat. Nam dui mi, tincidunt
       quis, accumsan porttitor, facilisis luctus, metus
     </p>
+
+     <div id="parent-root">
```

Comment:
This div is parent root element for both embedded chat and floating chatbox.
Since we can only create 2 root element using react when we have a parent root div.